doc/tutorials/transformer/question_answering.ipynb

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
<p>maybe ad a note on what is special about this model. also would be nice in the other notebooks where its missing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>